app/extensions/string_extensions.rb

```diff
@@ -36,7 +36,7 @@ class String
 
   # :stopdoc:
   # Disable cop in order to make code more readable
-  # rubocop:disable Layout/AlignHash
+  # rubocop:disable Layout/HashAlignment,Layout/AlignHash
```

Comment:
```ruby
  # rubocop:disable Layout/HashAlignment
```
The cop name was change in v.77.
See https://github.com/rubocop-hq/rubocop/issues/7077#issuecomment-546317645